### ISSUE TITLE
[BUG] "Modules Logs" shows an infinite amount of "Loading modules" entries

### ISSUE THREAD

**Issue by ChristophSchmidpeter** [ElderDrivers/EdXposed#818]
**什么情况/What happened?**
The log shows that activated modules are loaded in an infinite loop. This process appears to drain the battery quite rapidly.

Steps to reproduce:
- Enable Riru Core and Riru EdXposed (see versions below)
- Enable at least one EdXposed module
- Reboot
- Notice that the EdXposed's log gets littered with an infinite amount of log entries for each activated module like 
`01-31 12:23:03.341  5558  5558 I EdXposed-Bridge: Loading modules from /data/app/org.meowcat.edxposed.manager-AmMeG4FO9EVTuWRJLfakhQ==/base.apk
01-31 12:23:03.343  5558  5558 I EdXposed-Bridge:   Loading class org.meowcat.edxposed.manager.xposed.Enhancement`
as time passes by (see logs below).

**Xposed模块列表/Xposed Module List**
The log was captured with only _EdXposed Manager_ enabled. It is reproducible with any one or more EdXposed modules activated, though.

**Magisk模块列表/Magisk Module List**
Riru Core
Riru - EdXposed

**版本信息/Versions**

Android: 10.0

Magisk: 21.4

Riru: 23.4

EdXposed: 0.5.2.0_4672-master (debug)

**相关日志/Related Logs**
[EdXposed_Modules_20210131_122944.txt](https://github.com/ElderDrivers/EdXposed/files/5899610/EdXposed_Modules_20210131_122944.txt)
[EdXposed_Verbose_20210131_123016.txt](https://github.com/ElderDrivers/EdXposed/files/5899609/EdXposed_Verbose_20210131_123016.txt)

**Comment by yujincheng08** [comment]
Expected.

**Comment by ChristophSchmidpeter** [comment]
@yujincheng08

If I run the phone for some hours, each module according to logs gets loaded many thousands of times.
- Real debug info gets very hard to find among all the noise in the log.
- More importantly this obviously **wastes battery / CPU capacity** by a high degree.

How can this behaviour be as expected? Why should it be intended to reload each module over and over again in an infinite loop?

**Comment by yujincheng08** [comment]
because u apply all modules to all apps. Try use whitelist & module scope

**Comment by ChristophSchmidpeter** [comment]
@yujincheng08 
I see. Thanks for the info. So for each installed app on the phone a modules load entry is created per module, right?
I have now enabled the whitelist mode and it looks better. One thing I've noticed though, is that each time I reopen the EdXposed Manager app, new module load entries are created on reopening the app. Shouldn't the loading of modules by the backend be independent of opening the EdXposed Manager front end?
